website/server/models/user/schema.js

```diff
@@ -9,6 +9,8 @@ import {
 
 const Schema = mongoose.Schema;
 
+const INVALID_DOMAINS = ['habitica.com', 'habitrpg.com'];
```

Comment:
Makes sense
On Wed, Jun 15, 2016 at 12:12 PM Blade Barringer <notifications@github.com>
wrote:

> In website/server/models/user/schema.js
> <https://github.com/HabitRPG/habitrpg/pull/7659#discussion_r67204132>:
>
> > @@ -9,6 +9,8 @@ import {
> >
> >  const Schema = mongoose.Schema;
> >
> > +const INVALID_DOMAINS = ['habitica.com', 'habitrpg.com'];
>
> Full disclosure: I was going to convert this to an env variable while
> merging.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7659/files/2161cc5748870bd5ae48a18fa32f4ac8522b3f95#r67204132>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALz_85n0dbhP0mjSzqMJy6dMJfavDOBPks5qMDKCgaJpZM4I06jn>
> .
>